lib/pages/login_page.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_survey/pages/widgets/blur_background.dart';
+import 'package:flutter_survey/pages/widgets/dimmed_background.dart';
+import 'package:flutter_survey/resouces/assets.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class LoginPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoPageScaffold(
+      resizeToAvoidBottomInset: false,
+      child: Stack(
+        children: [
+          Container(
+            decoration: BoxDecoration(
+              image: DecorationImage(
+                image: AssetImage(Assets.bgLogin),
+                fit: BoxFit.cover,
+              ),
+            ),
+          ),
+          BlurBackground(),
+          DimmedBackground(
+            colors: [
+              Colors.black.withOpacity(0.2),
+              Colors.black.withOpacity(0.8),
+              Colors.black,
+            ],
+            stops: [0.0, 0.6, 1.0],
+          ),
+          Padding(
+            padding: EdgeInsets.all(24.0),
+            child: Column(
+              children: <Widget>[
+                Expanded(
+                  child: SvgPicture.asset(Assets.icNimbleLogo),
+                ),
+                _buildLoginForm(),
+                Expanded(
+                  child: SizedBox.shrink(),
```

Comment:
I'm curious why do we need this? 🤔

Reply:
@sleepylee to put a empty child into `Expanded` as I need to keep Expanded widget to keep the login form in the vertically center

Reply:
oh..if you just wanted to center them then maybe just add this to your `Column` widget no?
```
mainAxisAlignment: MainAxisAlignment.center,
crossAxisAlignment: CrossAxisAlignment.center,
```

Reply:
hmm, can't, I would like to keep the top logo in the center of the rest of the top area too 🤔 

<img src="https://user-images.githubusercontent.com/16315358/132206751-215abe83-70f1-4c6b-b21e-7f6b018ed323.png" width=320 />

Reply:
🤔  ok